kani-compiler/src/args.rs

```diff
@@ -4,6 +4,20 @@
 use strum_macros::{AsRefStr, EnumString, VariantNames};
 use tracing_subscriber::filter::Directive;
 
+#[derive(Debug, Default, Clone, Copy, AsRefStr, EnumString, VariantNames, PartialEq, Eq)]
+#[strum(serialize_all = "snake_case")]
+pub enum BackendOption {
+    /// Aeneas (LLBC) backend
+    Aeneas,
```

Comment:
This option should only be available if the compiler feature was enabled, right?

Reply:
Good point.